### ISSUE TITLE
LRCN_OP_YOLO

### ISSUE THREAD

**Issue by psnkhoma** [jaiduqdom/LRCN_OP_YOLO#2]
I have been trying to run the code to learn what it is doing. I got to step4 and got errors as if the code is incomplete. I have you tried of late to verify if this whole code runs? Is there an updated version of this implementation?

**Comment by jaiduqdom** [comment]
Hi,Could you send me a screenshot with your errors? Maybe some previous step is missing.Thanks
De: "psnkhoma" ***@***.***
A: "jaiduqdom/LRCN_OP_YOLO" ***@***.***
Cc: "Subscribed" ***@***.***
Fecha: Fri, 08 Mar 2024 13:04:33 -0800
Asunto: [jaiduqdom/LRCN_OP_YOLO] LRCN_OP_YOLO (Issue #2)


I have been trying to run the code to learn what it is doing. I got to step4 and got errors as if the code is incomplete. I have you tried of late to verify if this whole code runs? Is there an updated version of this implementation?

**Comment by psnkhoma** [comment]
Here is the screenshot where I encountered a problem.
![Screenshot from 2024-03-09 21-19-22](https://github.com/jaiduqdom/LRCN_OP_YOLO/assets/129693427/dbedb102-2ebd-4018-bdae-8c936f3ff749)

**Comment by jaiduqdom** [comment]
Hi,

   Can you try replacing the line

   DIRECTORIO_VIDEOS_STAIRS="/datos/NTURGBD/VIDEOS_50_120"

   with

   DIRECTORIO_VIDEOS_STAIRS="/datos/NTURGBD/VIDEOS"

   (in paso4)

   I think it was changed due to the last experiments with NTU-RGB-D.

   Kind regards,

   Jaime

   De "psnkhoma" ***@***.***
   Para "jaiduqdom/LRCN_OP_YOLO" ***@***.***
   Cc "jaiduqdom" ***@***.***,"Comment" ***@***.***
   Fecha Sat, 09 Mar 2024 12:26:17 -0800
   Asunto Re: [jaiduqdom/LRCN_OP_YOLO] LRCN_OP_YOLO (Issue #2)

   Here is the screenshot where I encountered a problem.
   Screenshot.from.2024-03-09.21-19-22.png (view on web)

**Comment by psnkhoma** [comment]
Thank you, i will try to make those changes. On another note, in the data_YOLO_OpenPose_Features.py there is a file named data_file_filtrado_shuffle.csv. Did you create this file separately or it is included in the downloaded folders. Here is the line: 
with open(os.path.join('/datos','NTURGBD','data_file_filtrado_shuffle.csv'), 'r') as fin:
I am trying to use the STAIR dataset only and not the NTURGBD and NTURGBD-120. Thank you for assisting, am sorry to bother you again.

**Comment by jaiduqdom** [comment]
Yes. That file is generated in paso4 file. The code was initially prepared for the STAIR dataset but some changes were made to evaluate NTURGB. For that reason there are some places where the names of the files could be NTURGB instead of STAIR. But pointing to the right STAIR folder and checking the names of the intermediate files should be ok to run it. Hope it runs succesfully.... but take into account that the generation of intermediate files and the training could take several days with GPU.
De: "psnkhoma" ***@***.***
A: "jaiduqdom/LRCN_OP_YOLO" ***@***.***
Cc: "jaiduqdom" ***@***.***,"Comment" ***@***.***
Fecha: Wed, 13 Mar 2024 08:01:03 -0700
Asunto: Re: [jaiduqdom/LRCN_OP_YOLO] LRCN_OP_YOLO (Issue #2)


Thank you, i will try to make those changes. On another note, in the data_YOLO_OpenPose_Features.py there is a file named data_file_filtrado_shuffle.csv. Did you create this file separately or it is included in the downloaded folders. Here is the line:
with open(os.path.join('/datos','NTURGBD','data_file_filtrado_shuffle.csv'), 'r') as fin:
I am trying to use the STAIR dataset only and not the NTURGBD and NTURGBD-120. Thank you for assisting, am sorry to bother you again.
